### DISCUSSION TITLE
is there a reason why sk vs-code plugin would output different / better response than python sk inline code?

### DISCUSSION THREAD

**Discussion by selfishark** [microsoft/semantic-kernel#5311] [Q&A] [answered]
Is there any reason why an ai model (for instance azure open ai gpt-4) will output an incomplete (with duplicate data) response when using inline code (semantic kernel plugin from directory) whereas when using the semantic kernel extension on vs-code the same prompt (from the plugin directory) output and clear and complete response without duplicates?

**Comment by madsbolaris** [comment, marked as answer]
They should behave the same. Can you confirm that you are using the same model? And can you share the code and prompt you are using to run the prompt using Semantic Kernel?
